openstates/wi/bills.py

```diff
@@ -350,7 +353,7 @@ def add_senate_votes(self, vote, url):
             vote.set_count(vote_type, vote_counts[vote_type])
 
         if name_counts != vote_counts:
-            self.error("Yes votes and number of Names doesn't match")
+            self.error("Names not extracted correctly")
             raise ValueError("Vote Count and number of Names don't match")
```

Comment:
Are `self.error` and `raise …` redundant?

Reply:
Looks like there are a couple cases further down in the code, too.

Reply:
I will delete self.error then!

Reply:
I personally think, we should throw error, as we will get notified for that, and hence we can act on it.